services/frontend/src/components/tasks/task-modal.js

```diff
@@ -189,7 +191,7 @@ class TaskModal extends React.Component {
                         )}
                         <DueDatePicker
                             entityId={_.get(this.state.task, 'tid')}
-                            value={_.get(this.state.task, 'dueDate')}
+                            value={isSubtask ? _.get(relatedAssignment, 'dueDate') : _.get(this.state.task, 'dueDate')}
```

Comment:
I tried making a new subtask and it has the correct default value but then I can't change it from that. I think we need to set the default somewhere else

Reply:
I'll look into it